test/jsdocUtils.js

```diff
@@ -10,6 +10,12 @@ describe('jsdocUtils', () => {
         it('returns the primary tag name', () => {
           expect(jsdocUtils.getPreferredTagName({}, 'jsdoc', 'return')).to.equal('returns');
         });
+        it('works with the constructor tag', () => {
+          expect(jsdocUtils.getPreferredTagName({}, 'jsdoc', 'constructor')).to.equal('class');
+        });
+      });
+      it('works with tags that class with prototype properties', () => {
+        expect(jsdocUtils.getPreferredTagName({}, 'jsdoc', 'toString')).to.equal('toString');
```

Comment:
Without the `src` change, this test fails with `expected [Function Object] to equal 'class'`.